scripts/convert_lit_checkpoint.py

```diff
@@ -196,6 +196,20 @@ def maybe_unwrap_state_dict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, t
     return lit_weights.get("model", lit_weights)
 
 
+def maybe_raise_finetune_warning(lit_weights: Dict[str, torch.Tensor], checkpoint_dir: Path) -> None:
+    weight_names = set([wk.split(".")[-1] for wk in lit_weights.keys()])
+    # adapter
+    if "gating_factor" in weight_names:
+        raise Exception(f"Converting models finetuned with adapter not yet support")
+    # adapter v2
+    elif "adapter_bias" in weight_names:
+        raise Exception(f"Converting models finetuned with adapter_v2 not yet support")
+    # LoRA or QLoRA
```

Comment:
LoRA or QLoRA can be supported as long as the adapter weights have been merged into the checkpoint. The error message should describe this

Reply:
Updated in [1d54da0](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/1d54da0465d7d2e97c8d17ef770ff31c1c8220ce)